bench/Main.hs

```diff
@@ -185,6 +186,59 @@ main = do
                 !range = (1, n - 1)
             in pureUniformRBench @Natural range sz
           ]
+        , bgroup "floating"
+          [ bgroup "IO"
+            [ bench "uniformFloat01M" $ nfIO $ runStateGenT_ (mkStdGen 1337) $ \g ->
+                replicateM_ sz $ do !_ <- uniformFloat01M g
```

Comment:
Is it possible to make this more generic, i.e. to run all benchmarks with `runStateGenT_` rather than just this specific one? I would really like us to have a full benchmark matrix.

Reply:
Surely it's possible. I however would prefer to do it in separate PR in order to one thing at a time

Reply:
Cool. Filed https://github.com/idontgetoutmuch/random/issues/161.